### PR TITLE
ci: delete virbr0 interface for vagrant during cleanup

### PR DESCRIPTION
**Pull request by lebauce**
skip skydive-go-fmt
skip skydive-functional-tests-backend-elasticsearch
skip skydive-functional-tests-backend-orientdb
skip skydive-scale-tests
skip skydive-cdd-overview-tests
skip skydive-unit-tests
skip skydive-compile-tests
skip skydive-k8s-tests
skip skydive-ppc64le-tests
skip skydive-functional-tests-ebpf
skip skydive-functional-hw-tests

### CODE REVIEW

**Comment by lebauce** [comment]
run skydive-vagrant-tests

**Comment by lebauce** [comment]
run skydive-vagrant-tests

**Comment by lebauce** [comment]
run skydive-vagrant-tests

**Comment by lebauce** [comment]
run skydive-vagrant-tests

**Comment by lebauce** [comment]
run skydive-vagrant-tests

File: scripts/ci/cleanup.sh [hunk 1]
```diff
@@ -62,22 +62,23 @@ EOF
...
+  virsh net-destroy vagrant0
+  virsh net-destroy vagrant-libvirt
+  systemctl restart libvirtd
+
   rm -rf /tmp/skydive_agent* /tmp/skydive-etcd
   rm -rf /var/lib/jenkins/.vagrant.d/tmp
 
   # time to restart services
   sleep 8
+  ip l del virbr0
```
Comment (hunchback) [inline scripts/ci/cleanup.sh]: if you are attempting to remove virbr0 should you not use:

```
ip link set virbr0 down
brctl delbr virbr0
```

Also maybe this should be placed after https://github.com/skydive-project/skydive/pull/1707/files#diff-82887418280c1fded8efe3a50691b94dR73

Comment (lebauce) [reply scripts/ci/cleanup.sh]: Why use `brctl` ? `virbr0` is created by libvirtd so we need to wait a few seconds before deleting it

Comment (hunchback) [reply scripts/ci/cleanup.sh]: i used this link as reference: https://www.google.co.il/amp/s/www.thegeekdiary.com/how-to-remove-virbr0-and-lxcbr0-interfaces-on-centos-rhel-5-and-rhel-7/amp/

File: contrib/ansible/roles/skydive_dev/tasks/virtualbox.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+- package:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - akmod-VirtualBox
+    - VirtualBox
+    - kernel-devel
+    - "kernel-devel-{{ ansible_kernel }}"
```
Comment (nplanel) [inline contrib/ansible/roles/skydive_dev/tasks/virtualbox.yml]: kernel-headers-{{ ....

**Comment by nplanel** [comment]
run skydive-vagrant-tests

**Comment by nplanel** [comment]
run skydive-vagrant-tests

**Comment by nplanel** [comment]
run skydive-vagrant-tests

**Comment by nplanel** [comment]
run skydive-vagrant-tests

**Pull request closed without merging**